cannon/mipsevm/program/patch.go

```diff
@@ -45,10 +45,6 @@ func PatchGo(f *elf.File, st mipsevm.FPVMState) error {
 			})); err != nil {
 				return fmt.Errorf("failed to patch Go runtime.gcenable: %w", err)
 			}
-		case "runtime.MemProfileRate":
```

Comment:
Need to decide on the final approach for clearing this.  We talked about a few options:
- Leaving as-is (prev implementation)
- Setting `runtime.MemProfileRate` in op-program (current approach in this PR)
- [Patching in an envar](https://github.com/ethereum-optimism/optimism/blob/4fd9c3917baf9c17d1642576d73eea53616cc440/cannon/mipsevm/program/patch.go#L71) to set memprofilerate (`GODEBUG=memprofilerate=0`)

Reply:
It's fine as-is in this PR. Maybe we can add a TODO to patch the envar for later. This way we don't have to remember to set `runtime.MemProfileRate` for every program (including those that aren't op-programs) we test.

Reply:
added some todos